components/dashboard/sections/expenses/HostCreateExpenseModal.tsx

```diff
@@ -127,7 +127,6 @@ const hostCreateExpenseMutation = gql`
 
 const SUPPORTED_EXPENSE_TYPES = omit(ExpenseType, [
   ExpenseType.UNCLASSIFIED,
-  ExpenseType.CHARGE,
   ExpenseType.SETTLEMENT,
   ExpenseType.FUNDING_REQUEST,
 ]);
```

Comment:
Perhaps it would be cleaner to get an array of the supported expenses and not omit types.

Reply:
True, it would prevent side-effects if introducing new types.